samples/stock/demo/area/demo.html

```diff
@@ -1,5 +1,5 @@
 
-<script src="https://code.highcharts.com/stock/highstock.js"></script>
+<script src="https://code.highcharts.com/highstock.js"></script>
```

Comment:
This change will break [the online demo](https://www.highcharts.com/demo/stock/area) because the URL is not working for the CDN.